templates/patterns/carousel/carousel.ui_patterns.yml

```diff
@@ -1,77 +1,60 @@
       preview: true
     with_controls:
       type: boolean
-      label: "With controls"
-      description: "With controls (default: false)."
+      label: 'With controls'
+      description: 'Enable the controls (prev and next buttons). False by default.'
       preview: true
-    prev_label:
-      type: select
-      label: "Previous label"
-      description: "The previous label value."
-      options:
-        1: "Previous"
-        2: "Previous slide"
-      preview: 1
-    next_label:
-      type: select
-      label: "Next label"
-      description: "The next label value."
-      options:
-        1: "Next"
-        2: "Next slide"
-      preview: 1
     with_indicators:
       type: boolean
-      label: "With indicators"
-      description: "With indicators (default: false)."
+      label: 'With indicators'
+      description: 'With indicators (default: false).'
       preview: true
     autoplay:
       type: boolean
-      label: "Disable autoplay"
-      description: "Disable auto slide movement."
+      label: 'Disable autoplay'
+      description: 'True by default. If set to false the carousel will not slide automatically.'
       preview: true
     autoinit:
       type: boolean
-      label: "Disable autoinit"
-      description: "Disable carousel autoinit on page load."
-      preview: false
+      label: 'initialize the carousel'
+      description: 'True by default. If set to false, the carousel will need to be instantiated via javascript.'
     disable_touch:
       type: boolean
-      label: "Disable touch"
-      description: "Disable carousel swipe on touch."
+      label: 'Disable touch'
+      description: 'Allow the carousel to swipe on touch. False by default.'
+      preview: false
+    dark:
+      type: boolean
+      label: 'Dark'
+      description: 'Enable the dark mode.'
       preview: false
   fields:
-    active_item:
-      type: "text"
-      label: "Index of the active item"
     slides:
       type: array
-      label: "Slides"
-      description: "Each slide is a image with src and alt, caption, caption classes and interval of slide movement."
+      label: 'Slides'
+      description: 'Each slide is an image, a caption and interval of slide movement.'
       preview:
-        - caption: first slide label
-          caption_classes: "class-1 class-2"
+        - caption: 'First slide label'
           interval: 0
           image:
-            - theme: "image"
-              uri: "https://images.unsplash.com/photo-1533282960533-51328aa49826?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1140&h=385&q=80"
-              alt: "alt image 1"
+            - theme: 'image'
+              uri: 'https://placeimg.com/1000/500/arch'
+              alt: 'alt image 1'
               attributes:
-                class: d-block w-100
-        - caption: second slide label
-          caption_classes: "class-1 class-2"
+                class: 'd-block w-100'
```

Comment:
These classes are mandatory or the carousel won't look nice. We need to pass them in the pattern template without having the user to pass them explicitly.